### PR TITLE
bengali translation added

### PR DESCRIPTION
**Pull request by 5hafayet**
translations for the Bengali language has been added and more translation will be added.

### CODE REVIEW

**Comment by Tienisto** [comment]
Mark this PR as draft because not finished.

**Comment by 5hafayet** [comment]
completed the translation. please review and let me know if anything required

File: assets/i18n/strings_bn.i18n.json [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+  },
+  "aboutPage": {
+    "title": "LocalSend সম্পর্কে"
+  },
+  "changelogPage": {
+    "title": "Changelog"
+  },
+  "aliasGenerator": {
+    "@info": "বিভিন্ন ভাষায় বিভিন্ন শব্দ থাকতে পারে, এটি 1:1 এর মতো নাও মিলতে পারে",
+    "adjectives": [
```
Comment (Tienisto) [inline assets/i18n/strings_bn.i18n.json:146]: Can the adjectives and fruits be combinated without modification? In some languages the adjective is dependent on the subjective (fruit in this case).

Secondly, how does the combination look like? Adjective + Fruit or Fruit + Adjective? You can lookup the latest English translation. There is a "combination" entry below the fruits.

Comment (5hafayet) [reply assets/i18n/strings_bn.i18n.json]: Extremely sorry for being late. I had some schoolwork.

Yes, the adjectives and fruits can be combined without modification and translation is not required also in that case. 

Adjective+fruit is okay.

please review the updated file and let me know if I can improve more.

**Pull request merged**